iceoryx_hoofs/container/include/iox/detail/vector.inl

```diff
@@ -359,13 +408,26 @@ inline bool vector<T, Capacity>::erase(iterator position) noexcept
         // AXIVION Next Line AutosarC++19_03-M5.0.9 : False positive. Pointer arithmetic occurs here.
         uint64_t index{static_cast<uint64_t>(position - begin())};
         uint64_t n{index};
-        while ((n + 1U) < size())
+        if constexpr (std::is_trivially_copyable<T>::value)
+        {
+            if constexpr (!(std::is_trivially_destructible<T>::value))
+            {
+                at_unchecked(n).~T();
+            }
+            uint64_t dim{size() - n - 1U};
```

Comment:
nitpick: in the `emplace` method you called this variable `dataLen`. Could you also rename `dim` to `dataLen` to keep it consistent?

Reply:
Yes, done!